synapse/telepath.py

```diff
@@ -1327,15 +1397,25 @@ async def openinfo(info):
         path = info.get('path')
         name = info.get('name', path[1:])
 
+        if port is None:
+            port = 27942
```

Comment:
this bit failed for me since aha is listening on 27492

Reply:
That would be expected if we're not providing port numbers for listeners.

Reply:
On its own it gives a long needed default port for things to listen on & connect to.